utils/vlog/tests/dsp_in_registered/golden.pb_type.xml

```diff
@@ -5,262 +5,266 @@
-      <port from="dsp_in_registered" name="out[43]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[44]" type="input"/>
-      <port from="dsp_in_registered" name="out[44]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[45]" type="input"/>
-      <port from="dsp_in_registered" name="out[45]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[46]" type="input"/>
-      <port from="dsp_in_registered" name="out[46]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[47]" type="input"/>
-      <port from="dsp_in_registered" name="out[47]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[48]" type="input"/>
-      <port from="dsp_in_registered" name="out[48]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[49]" type="input"/>
-      <port from="dsp_in_registered" name="out[49]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[50]" type="input"/>
-      <port from="dsp_in_registered" name="out[50]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[51]" type="input"/>
-      <port from="dsp_in_registered" name="out[51]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[52]" type="input"/>
-      <port from="dsp_in_registered" name="out[52]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[53]" type="input"/>
-      <port from="dsp_in_registered" name="out[53]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[54]" type="input"/>
-      <port from="dsp_in_registered" name="out[54]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[55]" type="input"/>
-      <port from="dsp_in_registered" name="out[55]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[56]" type="input"/>
-      <port from="dsp_in_registered" name="out[56]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[57]" type="input"/>
-      <port from="dsp_in_registered" name="out[57]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[58]" type="input"/>
-      <port from="dsp_in_registered" name="out[58]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[59]" type="input"/>
-      <port from="dsp_in_registered" name="out[59]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[60]" type="input"/>
-      <port from="dsp_in_registered" name="out[60]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[61]" type="input"/>
-      <port from="dsp_in_registered" name="out[61]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[62]" type="input"/>
-      <port from="dsp_in_registered" name="out[62]" type="output"/>
-    </direct>
-    <direct>
-      <port name="out[63]" type="input"/>
-      <port from="dsp_in_registered" name="out[63]" type="output"/>
-    </direct>
+    <direct input="dff[0].q" name="dsp_combinational_a_0" output="dsp_combinational.a[0]"/>
```

Comment:
This should be reverted right?

Reply:
yep, I rebased the whole PR on top of current master. It now uses new direct connections style